vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/dataview/ComboBoxListDataView.java

```diff
@@ -82,4 +84,9 @@ public Registration addItemCountChangeListener(
             ComponentEventListener<ItemCountChangeEvent<?>> listener) {
         return super.addItemCountChangeListener(listener);
     }
+
+    @SuppressWarnings("rawtypes")
+    private Query getQuery() {
+        return dataCommunicator.buildQuery(0, Integer.MAX_VALUE);
```

Comment:
Just noting that I think this needs to be then fixed for https://github.com/vaadin/flow/issues/8655 to take the component stored filter into account

Reply:
Yes, that's true. But let's skip it for this PR, right?